packages/snap-preact-components/src/components/Organisms/MobileSidebar/MobileSidebar.tsx

```diff
@@ -0,0 +1,219 @@
+import { ComponentProps, StylingCSS } from '../../../types';
+import { Slideout, SlideoutProps } from '../../Molecules/Slideout';
+import { defined, mergeProps } from '../../../utilities';
+import { SearchController } from '@searchspring/snap-controller';
+import { Sidebar, SidebarProps } from '../Sidebar';
+import { Button, ButtonProps } from '../../Atoms/Button';
+
+const CSS = {
+	toolbar: () =>
+		css({
+			'& .ss__mobile-sidebar__title': {
+				justifyContent: 'space-between',
+				flexDirection: 'row',
+				display: 'flex',
+
+				'& .ss__icon': {
+					cursor: 'pointer',
+				},
+			},
+
+			'& .ss__mobile-sidebar__slideout__button': {
+				cursor: 'pointer',
+			},
+
+			'& .ss__mobile-sidebar__cta-wrapper': {
+				justifyContent: 'space-around',
+				flexDirection: 'row',
+				display: 'flex',
+			},
+		}),
+};
+
+export const MobileSidebar = observer((properties: MobileSidebarProps): JSX.Element => {
+	const globalTheme: Theme = useTheme();
+
+	const defaultProps: Partial<MobileSidebarProps> = {
+		slideoutButtonText: 'Filters',
+		clearButtonText: 'Clear All',
+		applyButtonText: 'Apply Filters',
+		titleText: 'Filter Options',
+		displayAt: '650px',
+		iconClose: 'close-thin',
+	};
+
+	const props = mergeProps('mobileSidebar', globalTheme, defaultProps, properties);
+
+	const {
+		controller,
+		hideFacets,
+		hideFilterSummary,
+		hidePerPage,
+		hideTitle,
+		hideSortBy,
+		hideApplyButton,
+		hideCloseButton,
+		slideoutButtonText,
+		clearButtonText,
+		applyButtonText,
+		iconClose,
+		iconOpen,
+		titleText,
+		displayAt,
+		hideClearAllButton,
+		disableStyles,
+		className,
+		style,
+	} = props;
+
+	const styling: { css?: StylingCSS } = {};
+
+	if (!disableStyles) {
+		styling.css = [CSS.toolbar(), style];
+	} else if (style) {
+		styling.css = [style];
+	}
+
+	const subProps: MobileSidebarSubProps = {
+		slideout: {
+			// default props
+			controller,
+			displayAt: `(max-width: ${displayAt})`,
+			// global theme
+			...globalTheme?.components?.slideout,
+			// inherited props
+			...defined({
+				disableStyles,
+			}),
+			// component theme overrides
+			theme: props?.theme,
+		},
+		button: {
+			// default props
+			// global theme
+			...globalTheme?.components?.button,
+			// inherited props
+			...defined({
+				disableStyles,
+			}),
+			// component theme overrides
+			theme: props?.theme,
+		},
+		sidebar: {
+			// default props
+			hideTitle: true,
+			hideFacets: hideFacets,
+			hidePerPage: hidePerPage,
+			hideSortBy: hideSortBy,
+			hideFilterSummary: hideFilterSummary,
+			// global theme
+			...globalTheme?.components?.sidebar,
+			// inherited props
+			...defined({
+				disableStyles,
+			}),
+			// component theme overrides
+			theme: props?.theme,
+		},
+	};
+
+	const Content = (props: any) => {
+		const { toggleActive } = props;
+		return (
+			<div className="ss__mobile-sidebar__content">
+				{!hideTitle && (
+					<div className="ss__mobile-sidebar__title">
+						<span>{titleText}</span>
```

Comment:
I think we may want to have a classname on this element (and maybe use an H tag?) - eg: `ss__mobile-sidebar__header__title`